client/src/redux/actions/index.js

```diff
@@ -0,0 +1,33 @@
+import * as credentialsActions from './credentialsActions';
+import * as factsActions from './factsActions';
+import * as reportsActions from './reportsActions';
+import * as scansActions from './scansActions';
+import * as sourcesActions from './sourcesActions';
+import * as statusActions from './statusActions';
+import * as userActions from './userActions';
+
+const actions = {
+  credentials: credentialsActions,
+  facts: factsActions,
+  reports: reportsActions,
+  scans: scansActions,
+  sources: sourcesActions,
+  status: statusActions,
+  user: userActions
+};
+
+const reduxActions = { ...actions };
+
+export {
+  reduxActions,
+  actions,
+  credentialsActions,
+  factsActions,
+  reportsActions,
+  scansActions,
+  sourcesActions,
+  statusActions,
+  userActions
+};
+
```

Comment:
Is there a plan to import actions differently?

Reply:
So was looking to make it consistent, namespacing for everything; reducers, actions, constants. From there it's the convenience of simply going that's an `action` prefix and possible intellisense on your IDE/editor

Reply:
Per our side discussion, going to remove this `index.js` file and reserve it for a testing pr if it becomes necessary